homeassistant/components/bond/cover.py

```diff
@@ -46,6 +55,25 @@ def __init__(
         """Create HA entity representing Bond cover."""
         super().__init__(hub, device, bpup_subs)
 
+    @property
+    def supported_features(self) -> int:
+        """Flag supported features."""
+        supported_features = 0
+        if self._device.supports_open():
```

Comment:
Is it possible to assign `_attr_supported_features` in the constructor instead ?

Reply:
Yes, done.  That's much better since it doesn't have to re-create it every time state is written 👍